adapters/wagmi-connector/package.json

```diff
@@ -14,15 +14,18 @@
   },
   "scripts": {
     "build": "tsc && vite build",
-    "lint": "eslint src --ext .ts"
+    "lint": "eslint src --ext .ts",
+    "test": "vitest"
```

Comment:
`"test": "vitest run --coverage"`
Maybe increase coverage and add more tests. Looking forward to 85%+